src/course_bot/essay.clj

```diff
@@ -81,7 +82,18 @@
     :essay-feedback-saved "Ваш отзыв сохранен и будет доступен авторам эссе."
     :essay-feedback "Отзыв: "
     :feedback-on-your-essay "отзыв на ваше эссе "
-    :number-of-reviews-1 "Вы получили %d отзывов."}}})
+    :number-of-reviews-1 "Вы получили %d отзывов."
+    :duplicate-text "Вы отправили текст, который очень похож на текст другого студента. Пожалуйста, отправьте оригинальный текст."}}})
+
+(defn jaccard-similarity [s1 s2]
+  (let [set1 (set (clojure.string/split s1 #"\s+"))
```

Comment:
Will be good, if you remove very short and very ling words. E.g.: less than 3 (но, с...) and bigger then 12 (usually it will be a links). It should increase performances.